js-foundations.js

```diff
@@ -2,24 +2,60 @@
 // Coding practical questions
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
-
-
+function divNum (num){
+    if(num%3 === 0){
+        return num + " is divisble by three"
+    }else if(num%3 === 1){
+        return num + " is not divisble by three"
+    }else if(num%3 !==0){
+       return num + " is not divisble by three"
+    }
+}
+console.log(divNum(15));
+console.log(divNum(0));
+console.log(divNum(385));
+console.log(divNum(-7));
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
-
-
+var helloMe = {
+    firstName: "Jesus",
+    lastName: "Ortega",
+    hand: "right handed",
+    music: "rap/hip-hop",
+    getData: function() {return "Hello my name is " + this.firstName + this.lastName + " and I am " + this.hand + " and I listen to " + this.music + " music."}
+    
+}
+console.log(helloMe.getData());
 
```

Comment:
Looks great! This would be a great place to use string interpolation to clean up your code a bit.